app/components/badge.js

```diff
@@ -124,14 +131,7 @@ export default class Badge extends PureComponent {
                 {...this.panResponder.panHandlers}
                 onPress={this.handlePress}
             >
-                <View
-                    ref='badgeContainer'
-                    style={[styles.badge, this.props.style, {opacity: 0}]}
-                >
-                    <View style={styles.wrapper}>
```

Comment:
Move this View wrappers to renderText so that snapshot will get the shape of what we've written, excluding the `TouchableWithoutFeedback` (which has lots of properties that makes the snapshot a mess if included).